netlogo-gui/src/main/lab/gui/LabManager.scala

```diff
@@ -36,7 +37,11 @@ class LabManager(val workspace:        GUIWorkspace,
     protocols.clear()
   }
   private lazy val dialog = new ManagerDialog(this, dialogFactory, menuFactory)
-  def show() { dialog.update(); dialog.setVisible(true) }
+  def show() { 
+    Positioning.center(dialog, workspace.getFrame) //Set dialog position to center of main window
```

Comment:
This comment isn't necessary and should be removed. We want our comments to talk about *why* code is doing what it's doing, not just to describe behavior that is already clear from the code itself ([see this CodingHorror blog post](https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/)). Here, notice that this comment (which is a good description of what is going on) has most of the the same words as the line of code. This is a good sign! It means the code indicates clearly what it is doing without needing comments. If we were doing something unusual here, we might want a comment explaining why we were doing that, but in this case it seems obvious that we want to center the window because that offers a better experience for our users.